babynames.py

```diff
@@ -61,20 +81,29 @@ def create_parser():
     return parser
 
 
+def summary(text, filename):
+    with open(filename, "w+") as file:
+        for name in text:
+            file.write(name + "\n")
```

Comment:
assuming `text` is a list of names, instead of a for-loop, could also write here:
```file.write('\n'.join(text))```
Ideally the `text` variable would be named `name_list` or something like that, to help think of it as a list instead of a big string